packaging/systemd/flotta-agent.service

```diff
@@ -9,8 +9,6 @@ Type=oneshot
 RemainAfterExit=yes
 # Since home directory is customed, we need to create it manually with specific selinux labels to allow access to it.
 ExecStart=bash -c "mkdir -p /var/home"
-ExecStart=bash -c "semanage fcontext -a -e /home /var/home"
```

Comment:
this fails with error:
```
Error: Transaction test error:
  file /etc/selinux/targeted/contexts/files/file_contexts.subs from install of flotta-agent-0.1.0-3.fc35.x86_64 conflicts with file from package selinux-policy-targeted-35.3-1.20211019git94970fc.fc35.noarch
```
perhaps for ostree this can be mitigated by adding `|| true` not to fail for ostree?